python/tests/conftest.py

```diff
@@ -104,6 +107,11 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize("encoding_version", [EncodingVersion.V1] if only_v1 else list(EncodingVersion))
 
 
+@pytest.fixture(scope="session")
```

Comment:
Why is this session scoped? Could lead to horrible problems if a test mutates it